src/fu-util-bios-attr.c

```diff
@@ -36,6 +36,9 @@ fu_util_bios_attr_update_description(FwupdBiosAttr *attr)
 			/* TRANSLATORS: description of a BIOS setting */
 			new = _("BIOS updates delivered via LVFS or Windows Update");
 		}
+		/* Try to look it up from translations */
+	} else {
+		new = _(old);
```

Comment:
I'm not sure this is 100% correct; don't you want a call to `gettext()` instead?